### DISCUSSION TITLE
Unable to Select Duplicate Item in a DataGrid

### DISCUSSION THREAD

**Discussion by HobbesKo** [AvaloniaUI/Avalonia#8307] [Q&A]
I am binding an `ObservableCollection<string>` to a DataGrid column.

If the collection contains the same string more than once, I am unable to select the cell that contains the duplicate.
`SelectedIndex` will return the index of the first occurrence of the string, and not the index of the cell on which I actually clicked.

Is this just how it is or am I missing something?

**Comment by D0ctorWh0** [comment]
Having same problem with avalonia 10.17. Should it works like that or should I update avalonia version?

**Reply by HobbesKo** [reply]
I've not tried binding an `ObservableCollection<string>` to a DataGrid column since I posted this question.
I was told that the issue is likely to be a bug.

I've no idea whether it is fixed in later Avalonia versions.
